test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -1211,7 +1211,7 @@ class test_uct_cm_sockaddr_stress : public test_uct_cm_sockaddr {
         int index;
 
         index = get_ep_index(ep);
-        ASSERT_GE(index, 0);
+        ASSERT_TRUE(index >= 0);
```

Comment:
do you see this warning since different signedness?
but if I'm not mistaken, they both have to be signed integer

Reply:
later on in this function index is used to access a field in an array, so therefore it can't be a negative value.